components/21-atoms/heading/heading.twig

```diff
@@ -1,9 +1,9 @@
 {% if type == "overview-title" %}
-  <{{ heading|default('h1') }} class="overview-title">{{ heading_text }}</{{ heading|default('h1') }}>
+  <{{ heading|default('h1') }} class="overview-title {{ heading_class }}">{{ heading_text }}</{{ heading|default('h1') }}>
```

Comment:
Geeft die header class niet de indruk dat die er op alle headers moet staan?

Reply:
T is optioneel voor bv. een h2 met class h3, zoals in het design vaker voorkomt. Heb je graag dat ik er een {% if %} rond zet?

Reply:
Zolang we niet de indruk wekken dat die class **nodig** is op een heading, is het goed voor mij.